ocaml/typing/rec_check.ml

```diff
@@ -1192,13 +1192,24 @@ and case
        ----------------------------------------
        G - p; m[mp] |- (p (when g)? -> e) : m
     *)
```

Comment:
The text of this comment needs to be updated.

Reply:
shouldn't be necessary after addressing comment below?